geowebcache/mbtiles/src/main/java/org/geowebcache/mbtiles/layer/MBTilesLayer.java

```diff
@@ -352,4 +354,21 @@ public void setExpirationHeader(HttpServletResponse response, int zoomLevel) {
          * itself
          */
     }
+
+    @Override
+    public boolean supportsTileJSON() {
+        return true;
+    }
+
+    @Override
+    public TileJSON getTileJSON() {
+
```

Comment:
Why an empty line here? :D

Reply:
No idea (Didn't notice that) maybe hitting Enter on a different windows whilst the focus was actually on the IDE :D 
Removing it.